### ISSUE TITLE
0.20.0 fails to build: 

### ISSUE THREAD

**Issue by yurivict** [MDAnalysis/mdanalysis#2338]
```
Attempting to autodetect OpenMP support... Did not detect OpenMP support.
No openmp compatible compiler found default to serial build.
Will not attempt to use Cython.
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "setup.py", line 547, in <module>
    exts, cythonfiles = extensions(config)
  File "setup.py", line 324, in extensions
    if cython_linetrace:
NameError: name 'cython_linetrace' is not defined
```

py36-cython-0.29_1 is installed.

FreeBSD 12 amd64

**Comment by richardjgowers** [comment]
Hi thanks for reporting this, yeah, this is a small mistake with that release, I’ll make a patch release for this soon.

**Comment by richardjgowers** [comment]
Ok this is now fixed in the 0.20.1 release @yurivict
